src/ray/core_worker/core_worker_process.cc

```diff
@@ -23,6 +28,33 @@ namespace {
 
 std::unique_ptr<CoreWorkerProcessImpl> core_worker_process;
 
+// Get out and error filepath for worker.
```

Comment:
Add a comment here and there and reference each other saying they need to be in sync.